autogen/docs/systemdynamics.md.mustache

```diff
@@ -312,6 +312,7 @@ a plot to watch the changes in sheep population.
 >
 >         to setup
 >           ca
+>           set-current-plot "populations"
```

Comment:
Did you consider putting the `set-current-plot` command in the `go` procedure right before the `system-dynamics-do-plot` call?  To me that seems like the safer place for it, to make sure it's always correct no matter what other commands are run.